### PR TITLE
SORD now has a chance to crit for massive damage.

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: SORD now has a chance to crit for massive damage.
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
there we go

**Comment by McBawbaggings** [comment]
I got a good smile
+1

**Comment by Arzion5** [comment]
Fun is now allowed. +1

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/obj/item/sord/attack(mob/living/M, mob/living/user)
+	if(prob(10))
+		M.adjustBruteLoss(1)
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/weaponry.dm:3]: M.apply_damage(1, BRUTE)

**Pull request merged**